tests/test_chebyshevBoundedField.py

```diff
@@ -428,6 +428,18 @@ def testArrayFit(self):
                                                                     array, weights, ctrl)
                 self.assertFloatsAlmostEqual(
                     outField2.getCoefficients(), coefficients, rtol=1E-7, atol=1E-7)
+                # If we make a matrix for the same points and use it to fit,
+                # we get coefficients in unspecified order, but there are
+                # (up to round-off error) the same as the nonzero entries of
+                # the 2-d coefficients arrays.
+                matrix = lsst.afw.math.ChebyshevBoundedField.makeFitMatrix(self.bbox, self.xFlat, self.yFlat,
+                                                                           ctrl)
+                coefficientsPacked, _, _, _ = np.linalg.lstsq(matrix, array)
+                coefficientsPacked.sort()
+                coefficientsFlat = coefficients.ravel()
```

Comment:
I think in this case you probably want `coefficients.flatten()` which makes a copy, while `coefficients.ravel()` makes a view and thus you will be changing them in the next line and while that's fine in the test it's probably not what you want.